awscli/formatter.py

```diff
@@ -91,10 +92,11 @@ def _format_response(self, command_name, response, stream):
         # the response will be an empty string.  We don't want to print
         # that out to the user but other "falsey" values like an empty
         # dictionary should be printed.
-        if response:
```

Comment:
After looking at this again, I think we can just change this to explicitly check for '{}'.

The original intent of this (years ago) was that any AWS operation that did not have an associated output shape should *not* generate any CLI output.  In older version of the CLI, some operations could generate either an empty dict or empty string.

However, with the switchover to the normalized model parser, anytime an operation does not have an output shape, botocore gives back a dictionary that only has the RequestMetadata key populated.  Once the formatter's here strip that key out, we're left with an empty dictionary.

So it should be safe to update this to only check for an empty dictionary.  This meets the constraint that anything without an output shape should not generate CLI output.

Does that work for you?